app/views/projects/show.html.erb

```diff
@@ -1,2 +1,19 @@
-<h1>Projects#show</h1>
-<p>Find me in app/views/projects/show.html.erb</p>
+<p id="notice"><%= notice %></p>
```

Comment:
remove notice , you have acces to it in every view due to the fact that it inherits form application.html.erb